### ISSUE TITLE
Android: Accessibility timeout setting

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30866]
## Requires API Proposal
This issue may require a new API, or a change to an existing API. An API proposal should be added and discussed before proceeding with implementation. The API proposal can be added in the comments of this issue or linked as a separate issue.

## Description
Android has a setting called "Time to take action (accessibility timeout)", which allows a user to choose how long any UI that auto-dismisses or auto-progresses should stay active for.  There is currently no way to query this setting.

## React Native version:
v0.63

## Expected Behavior
The AccessibilityInfo API should have a way to query this setting.

## Android Details
This setting can be queried from `AccessibilityManager.getRecommendedTimeoutMillis()` method.
https://developer.android.com/reference/android/view/accessibility/AccessibilityManager#getRecommendedTimeoutMillis(int,%20int)

**Comment by Naturalclar** [comment]
Seems like this has been worked on in PR #31063
